src/components/auditors.vue

```diff
@@ -116,23 +116,15 @@
             </tr>
 
             <tr>
-              <th scope="row"><a target="_blank" href="https://rhymetec.com/vciso/">Rhymetic</a></th>
```

Comment:
What happened here and to side channel? I wouldn't have expected a change to these other lines. And I don't see a comment in the PR description. What's going on here?